transaction/tests/assets/manifest_generator_error_blob_not_found_1.diag

```diff
@@ -0,0 +1,15 @@
+[1m[91merror[0m: [1mblob with hash 'f531e5e82dc195a7a9f6709f5ee048d9541d51c52904b7a76295a13ddfa377b1' not found[0m
```

Comment:
Can we add some non-English character test?

Reply:
Done